compiler/build.go

```diff
@@ -373,6 +378,18 @@ func makeErrsRelative(out []byte, workdir, appRoot, relwd string) []byte {
 		appPath := filepath.Join(appRoot, string(filename[len(prefix):]))
 		if rel, err := filepath.Rel(wdroot, appPath); err == nil {
 			lines[i] = append([]byte(rel), line[idx:]...)
+
+			// If this is an encore generated code file, let's grab the surrounding source code
+			if strings.Contains(rel, "__encore_") {
```

Comment:
This is to make it _much_ easier to debug compile issues while iterating on the code generator. I debated making it visible for all compile errors, but decided against it as we currently don't rewrite the users's code, so the line numbers we report _should_ align with their IDE's codebase